clash-prelude/src/Clash/Explicit/DDR.hs

```diff
@@ -159,7 +163,7 @@ ddrOut# clk rst i0 xs ys =
     -- That is why we drop the first value of the stream.
     let (_ :- out) = zipSig xs' ys' in out
   where
-    xs' = register clk rst i0 xs
-    ys' = register clk rst i0 ys
+    xs' = register clk rst (error "ddrOut#: bug in Clash") xs
+    ys' = register clk rst (deepErrorX "ddrOut: initial value undefined") ys
```

Comment:
Interesting change no. 2: First value is undefined. I'm not sure whether we need to add powerup values to the DDR primitives or not.